av/filter/graph.pyx

```diff
@@ -196,7 +196,11 @@ cdef class Graph(object):
 
     def push(self, frame):
 
-        if isinstance(frame, VideoFrame):
+        if frame is None:
+            contexts = self._context_by_type.get('buffer', [])
+            if len(contexts) == 0:
```

Comment:
I'm not to sure about this, if we receive None, we really have no way of knowing whether it's for audio or for video.. Would it make sense to flush both?

Reply:
@jlaine I agree; we can't know. I chose this based on the assumption that `graph.push` and `graph.pull`  are only meant for use with only one source and one sink total. Perhaps I misunderstood? 

If there are multiple sources of the same kind, `push` will raise a ValueError

https://github.com/PyAV-Org/PyAV/blob/95b4f818c0b53c18af8a1ae0d5d9044df1586545/av/filter/graph.pyx#L206-L207

and similarly pull will raise if there is more than one sink in total:

https://github.com/PyAV-Org/PyAV/blob/95b4f818c0b53c18af8a1ae0d5d9044df1586545/av/filter/graph.pyx#L216-L218

That said, I'm happy to push None to both contexts instead if there are multiple, or just leave pushing of None out of graphs (you can still use the context directly) if you think that is cleaner. Let me know.

Reply:
OK you're right, this "push" method only handles a single buffer. Maybe use the following:

`contexts = self._context_by_type.get('buffer', [])  + self._context_by_type.get('abuffer', [])`

That way it will bomb in the same way as for AudioFrame / VideoFrame?